packages/search/src/search.tsx

```diff
@@ -443,7 +449,71 @@ export class Search extends React.Component< Props, State > {
 		}
 
 		return null;
-	}
-}
+	}, [ hideClose, keyword, isOpen, closeSearch, __, instanceId, closeListener ] );
+
+	return (
+		<div dir={ dir } className={ searchClass } role="search">
+			<Spinner />
+			{ openIconSide === 'left' && renderOpenIcon() }
+			<form className={ fadeClass } action="." onSubmit={ handleSubmit }>
+				<input
+					type="search"
+					id={ 'search-component-' + instanceId }
+					autoFocus={ autoFocus } // eslint-disable-line jsx-a11y/no-autofocus
+					aria-describedby={ describedBy }
+					aria-label={ inputLabel ? inputLabel : __( 'Search', __i18n_text_domain__ ) }
+					aria-hidden={ ! isOpenUnpinnedOrQueried }
+					className={ inputClass }
+					placeholder={ placeholder }
+					role="searchbox"
+					value={ searchValue }
+					ref={ searchInput }
+					onChange={ onChange }
+					onKeyUp={ onKeyUp }
+					onKeyDown={ onKeyDown }
+					onMouseUp={ onClick }
+					onFocus={ onFocus }
+					onBlur={ onBlur }
+					disabled={ disabled }
+					autoCapitalize="none"
+					dir={ dir }
+					maxLength={ maxLength }
+					minLength={ minLength }
+					{ ...autocorrectProps }
+				/>
+				{ renderStylingDiv() }
+			</form>
+			{ shouldRenderRightOpenIcon ? renderOpenIcon() : renderCloseButton() }
+		</div>
+	);
+};
+
+export const Search = React.forwardRef( InnerSearch );
+Search.defaultProps = {
+	autoFocus: false,
+	compact: false,
+	delaySearch: false,
+	delayTimeout: SEARCH_DEBOUNCE_MS,
+	describedBy: undefined,
+	dir: undefined,
+	disableAutocorrect: false,
+	disabled: false,
+	fitsContainer: false,
+	hideClose: false,
+	hideOpenIcon: false,
+	defaultIsOpen: false,
+	onClick: noop,
+	onKeyDown: noop,
+	onSearchChange: noop,
+	onSearchOpen: noop,
+	onSearchClose: noop,
+	openIconSide: 'left' as const,
+	//undefined value for overlayStyling is an optimization that will
+	//disable overlay scrolling calculation when no overlay is provided.
+	overlayStyling: undefined,
+	pinned: false,
+	recordEvent: noop,
+	searching: false,
+};
 
 export default withI18n( Search );
```

Comment:
It's better to use the `useI18n` hook now when the component is functional. The `__` function is then not mixed into the props, but is purely internal.